### ISSUE TITLE
Vercel version

### ISSUE THREAD

**Issue by KallkaGo** [teppyboy/Genshin-Card#21]
I referred to the implementation of your axios branch, and corrected the data misalignment, and has successfully deployed it to vercel. If you need it, you can refer to my implementation

**Comment by KallkaGo** [comment]
here is link [genshin card](https://github.com/KallkaGo/Genshin-Card)

**Comment by teppyboy** [comment]
Thanks, although i haven't used this for a while now lol (because this originally isnt my code and its too spaghetti for me to maintain). For now I'll update the readme to link to your repo for deploying in Vercel

**Comment by teppyboy** [comment]
Done (https://github.com/teppyboy/Genshin-Card/commit/424a9189deadfd0bca26e135ec7280434545588e), although your version is for the Chinese server I assume because it doesn't work with my overseas hoyolab UID lol. I'll keep this issue open because this repo is meant for overseas version of hoyolab

**Comment by KallkaGo** [comment]
yeah,my version is for the cn server,I don't know about hoyolab, maybe I just need to replace the requested address. I will do some tests and deploy a hoyolab version when I have time lol

**Comment by KallkaGo** [comment]
After a night of trying, I have completed the construction of the HOYOLAB version [HOYOLAB-Version](https://hoyolabgenshincard.vercel.app/)
The specific implementation is in my hoyolab branch of Genshin Card [repo](https://github.com/KallkaGo/Genshin-Card/tree/hoyolab)
Hope this helps you lol

**Comment by teppyboy** [comment]
nice, it works well for me, i'll add it to readme too lol

**Comment by KallkaGo** [comment]
I have merged your PR and redeployed it to vercel. I will update the README tomorrow to tell those who want to deploy it to their own vercel.

**Comment by teppyboy** [comment]
yep, thank you for porting the project to Vercel ❤️
